index.html

```diff
@@ -87,112 +133,114 @@ <h3>LET'S CONNECT</h3>
         />
         <div class="project-section">
```

Comment:
Kindly notice that the left margin does not match the right margin in your mobile design for the work cards. Please fix that 🤗

Your design:
![image](https://user-images.githubusercontent.com/73714615/186908025-fee07910-2952-4e7a-a9d1-b87966842058.png)

Figma design:
![image](https://user-images.githubusercontent.com/73714615/186908101-147f96bf-24c0-47c8-9421-966550ac705e.png)